community/server/src/main/java/org/neo4j/server/modules/AuthorizationModule.java

```diff
@@ -49,11 +52,18 @@ public AuthorizationModule( WebServer webServer, Supplier<AuthManager> authManag
     @Override
     public void start()
     {
+        final Filter authorizationFilter;
+
         if ( config.get( GraphDatabaseSettings.auth_enabled ) )
         {
-            final AuthorizationFilter authorizationFilter = new AuthorizationFilter( authManagerSupplier, logProvider, uriWhitelist );
-            webServer.addFilter( authorizationFilter, "/*" );
+            authorizationFilter = new AuthorizationEnabledFilter( authManagerSupplier, logProvider, uriWhitelist );
         }
+        else
+        {
+            authorizationFilter = new AuthorizationDisabledFilter();
```

Comment:
To always add the http request wrapper which is now used to get the access mode for starting transactions.